### ISSUE TITLE
Use a CDN for audio (and other) files

### ISSUE THREAD

**Issue by kookster** [PRX/cms.prx.org#112]
feeder will handle this for podcast files, but not for audio on prx

**Comment by cavis** [comment]
@kookster I thought we were already using cloudfront for the upload S3 bucket.  Is that not the case?

**Comment by cavis** [comment]
Also - duplicate #89?

**Comment by kookster** [comment]
@cavis yeah, these are duplicates, and no, we don't use a CDN for audio on prx.org, the cloudfront on the bucket was for upload, not download
